src/configuration/locales/en/translations.json

```diff
@@ -1,3 +1,33 @@
 {
-  "Take the survey!": "Take the survey!"
+  "...Loading": "...Loading",
+  "Beekeeper": "Beekeeper",
+  "Beginner": "Beginner",
+  "Billed: {{value}}": "Billed: {{value}}",
+  "Description {{description}}:": "Description: {{description}}",
+  "Experience: {{experience}}": "Experience: {{experience}}",
+  "Expert": "Expert",
+  "FreeSpace": "Free space available",
+  "Garden": "Garden",
+  "HAPPYCULTEUR © COPYRIGHT 2019. ALL RIGHTS RESERVED.": "Happyculteur © COPYRIGHT 2019. ALL RIGHTS RESERVED.",
```

Comment:
Sure about the `©` char? Don't you want to only keep it in the translation?

Reply:
Key should reflect the sentence, will be easier to update it this way